ctapipe/containers.py

```diff
@@ -984,6 +984,7 @@ class MuonEfficiencyContainer(Container):
     impact_x = Field(nan * u.m, "impact parameter x position")
     impact_y = Field(nan * u.m, "impact parameter y position")
     optical_efficiency = Field(nan, "optical efficiency muon")
+    fit_convergence = Field(nan, "convergence of the fit")
```

Comment:
I'd calls this, in line with other such definitions, `is_valid`.

We could also add other fields with more specific information, e.g. using these fields:

```
minuit.fmin.has_parameters_at_limit
minuit.fmin.has_valid_parameters
minuit.fmin.is_valid
```

and the likelihood value is available as:

```
minuit.fval
```

While we are at it, we could also store the uncertainties:

```
minuit.errors
```

Reply:
Checking the minuit docs, `has_valid_parameters` and `is_valid` are aliases.

So we could just stick with `is_valid` and `parameters_at_limit`.

Reply:
Default should also be `False`, not `nan`

Reply:
Thanks, implemented

Reply:
You don't yet fill the `likelihood_value` and the `parameters_at_limit` fields